### PR TITLE
Change the og:image in the engineering team page

### LINKED ISSUE #1451: Change the og:image in the engineering team page
**Is your feature request related to a problem? Please describe.**
Currently, the og:image in the engineering team page is a scholarx logo

image

**Describe the solution you'd like**
Change it to the hero image in the engineering team page

image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1451 

## Goals

Changing the og:image in the engineering team page

## Approach

Changed the og:image in the engineering team page

### Screenshots

image

  
### Preview Link

https://pr-1458-sef-site.surge.sh/engineering-team.html

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: engineering-team.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <meta name="description" content="The engineering team is the backbone of the SEF community. This team is saturated with potent young talent who engage in various challenges when it comes to contribution.">
         <meta name="author" content="Sustainable Education Foundation">
-        <meta property="og:image" content="assets/img/brand/scholarx.png">
+        <meta property="og:image" content="assets/img/deve-page-hero.png">
```
Comment (anjula-sack) [inline engineering-team.html]: Let's rename the image to `engineering-page-hero-image.png`

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply engineering-team.html]: Sure

Comment (YoshithaRathnayake) [reply engineering-team.html]: Done! @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Thank you!

**Pull request merged**